<com.ccsi.taxiapp.premium.activity.MapActivity$WebServiceTask: com.ccsi.taxiapp.premium.data.WebResponse doInBackground(java.lang.Void[])> --> $d0 = virtualinvoke $r17.<android.location.Location: double getLatitude()>()
<com.ccsi.taxiapp.premium.activity.MapActivity$WebServiceTask: com.ccsi.taxiapp.premium.data.WebResponse doInBackground(java.lang.Void[])> --> r2 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r0 := @parameter0: java.lang.Exception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> r2 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> staticinvoke <android.util.Log: int e(java.lang.String,java.lang.String)>("4MyCabPremium", "Error in Common.logError")
--------------------------------------------------------

<com.ccsi.taxiapp.premium.util.MyLocation: void getCurrentAddress(android.content.Context,android.os.Handler)> --> $d1 = virtualinvoke $r5.<android.location.Location: double getLongitude()>()
<com.ccsi.taxiapp.premium.util.MyLocation: void getCurrentAddress(android.content.Context,android.os.Handler)> --> r3 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r0 := @parameter0: java.lang.Exception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> r2 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> staticinvoke <android.util.Log: int e(java.lang.String,java.lang.String)>("4MyCabPremium", "Error in Common.logError")
--------------------------------------------------------

<com.ccsi.taxiapp.premium.activity.MapActivity: void setMapZoom()> --> $d0 = virtualinvoke r2.<android.location.Location: double getLatitude()>()
<com.ccsi.taxiapp.premium.activity.MapActivity: void setMapZoom()> --> r11 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r0 := @parameter0: java.lang.Exception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r1 = virtualinvoke $r0.<java.lang.Exception: java.lang.String toString()>()
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> r2 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> staticinvoke <android.util.Log: int e(java.lang.String,java.lang.String)>("4MyCabPremium", "Error in Common.logError")
--------------------------------------------------------

<com.ccsi.taxiapp.premium.util.MyLocation: void getCurrentAddress(android.content.Context,android.os.Handler)> --> $d0 = virtualinvoke $r5.<android.location.Location: double getLatitude()>()
<com.ccsi.taxiapp.premium.util.MyLocation: void getCurrentAddress(android.content.Context,android.os.Handler)> --> r3 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r0 := @parameter0: java.lang.Exception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> staticinvoke <android.util.Log: int e(java.lang.String,java.lang.String)>("4MyCabPremium", $r1)
--------------------------------------------------------

<com.ccsi.taxiapp.premium.util.MyLocation: void updateLoc(android.location.Location,java.lang.String)> --> $d0 = virtualinvoke $r1.<android.location.Location: double getLatitude()>()
<com.ccsi.taxiapp.premium.util.MyLocation: void updateLoc(android.location.Location,java.lang.String)> --> r3 := @caughtexception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> $r0 := @parameter0: java.lang.Exception
<com.ccsi.taxiapp.premtest.Common: void logError(java.lang.Exception,java.lang.String)> --> staticinvoke <android.util.Log: int e(java.lang.String,java.lang.String)>("4MyCabPremium", $r1)
--------------------------------------------------------

